packages/core/src/api/blockManipulation/commands/replaceBlocks/replaceBlocks.ts

```diff
@@ -39,12 +39,12 @@ export function replaceBlocks<
     editor,
     blocksToRemove,
     (node, pos, tr, removedSize) => {
-      if (node.attrs.id === idOfFirstBlock) {
+      if (blocksToInsert.length > 0 && node.attrs.id === idOfFirstBlock) {
```

Comment:
is it easy to add a unittest for this one? Also, do you know / understand why we need the + 1?

Reply:
So I dug deeper into this and turns out that coincidentally, `removeBlocks` and `replaceBlocks` were both working despite using positions that were too wide by 1 in `tr.delete`.